templates/publisher/list.html

```diff
@@ -0,0 +1,128 @@
+{% extends 'base_layout.html' %}
+
+{% block meta_description %}{% endblock meta_description %}
+{% block meta_copydoc %}{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip is-shallow p-strip--grey">
+  <div class="u-fixed-width">
+    <h1 class="p-heading--2">My charms and bundles</h1>
+  </div>
+</section>
+<section class="p-strip u-extra-space is-shallow">
+  <div class="u-fixed-width">
+    <ul class="p-inline-list--stretch is-navigation u-no-margin--bottom">
+      <li class="p-inline-list__item">
+        <nav class="p-tabs" aria-label="Charms and bundles navigation">
+          <ul class="p-tabs__list" role="tablist">
+            <li class="p-tabs__item" role="presentation">
+              <a href="/charms" class="p-tabs__link" tabindex="0" role="tab" aria-controls="charms" {% if request.path.endswith('charms') %}aria-selected="true" {% endif %}>Charms</a>
```

Comment:
You should be able to replace `request.path.endswith('charms')` with `page_type`

Reply:
done